File: CS/RegistrationForm.Lesson3/ViewModel/MainViewModel.cs
using DevExpress.Mvvm.DataAnnotations;
using DevExpress.Mvvm.POCO;

namespace RegistrationForm.ViewModel {
    [POCOViewModel]
    public class MainViewModel {
        public static MainViewModel Create() {
            return ViewModelSource.Create(() => new MainViewModel());
        }
        protected MainViewModel() { }
    }
}

File: VB/RegistrationForm.Lesson3/ViewModel/MainViewModel.vb
Imports DevExpress.Mvvm.DataAnnotations
Imports DevExpress.Mvvm.POCO

Namespace RegistrationForm.ViewModel
    <POCOViewModel> _
    Public Class MainViewModel
        Public Shared Function Create() As MainViewModel
            Return ViewModelSource.Create(Function() New MainViewModel())
        End Function
        Protected Sub New()
        End Sub
    End Class
End Namespace